portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,60 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+// Function that shows or hides the list of courses
+// for a particular semester upon user click
+function toggleCourseList(listObject) {
+    let semester = listObject.id;
+    let semCourseList = "courselist" + semester;
+    let lst = document.getElementById(semCourseList);
+    if(lst.style.display === "none" || lst.style.display === "")
+    {
```

Comment:
nit: JavaScript if-else statements generally follow the style of 
if (condition) {
    <code>
} else {
    <code>
}
i.e. the { follows the condition on the same line.